javascript/react-js/react-introduction.md

```diff
@@ -214,6 +214,10 @@ If you want to get a better understanding of how create-react-app works and whic
 
 Often React testing is done with `react-testing-library`. Taking your learnings from the Battleship project, you should apply these concepts going forward in all your future projects, even with React.
 
+### React Developer Tools
+
+React Developer Tools is an extension to the browser developer tools which allows you to inspect React components that build the page. It helps you to check and edit the React component tree as well as props, state, and hooks for each component. [This article](https://www.pluralsight.com/guides/debugging-components-with-react-developer-tools) and [this video](https://www.youtube.com/watch?v=rb1GWqCJid4) will show you how to install React Developer Tools and some basic debugging you can do with it.
```

Comment:
In all honesty, I hesitate to point people to both an article they need to read and a video they need to watch for this. Reviewing the two resources I feel like the article is a better resource and contains more information. I'd like to see the video removed here.

If you think that the video is a better resource I can understand that viewpoint, but I would still want to limit the resources linked here to one. If you want to keep the video I would want to see the linked article moved to the additional resources section.

Reply:
Understood.